lib/Cro/WebApp/Template/AST.pm6

```diff
@@ -318,11 +325,21 @@ my constant %escapes = %(
     "'" => '&apos;',
 );
 
-sub escape-text(Str() $text) {
+multi escape-text(Nil $text, $file, $line) {
+    warn "An expression at $file:$line compiled into Nil";
+    ''
+}
+
+multi escape-text(Str() $text, *@_) {
```

Comment:
From a performance point of view, it's probably better to have `Str() $text, $, $)` as the signature, as the `*@_` actually creates an `Array`, while the bare `$` indicates something to be accounted for during the dispatch process, but not something that should actually bound.

Reply:
This, and type them as `Mu`, so there's no type check code emitted (it'll likely be all the same after specialization, but helps an iota until then).